docs/datamodel/links.rst

```diff
@@ -277,6 +277,29 @@ table <https://en.wikipedia.org/wiki/Associative_entity>`_, whereas a
 As a result, single links are marginally more efficient. Generally ``single``
 links are recommended when modeling 1:N relations.
 
+Another decision often faced by EdgeDB users is whether to use a ``multi``
+link on the linking object or a backlink on the other (the linked object).
+A general rule of thumb in this case is as follows.
+
+Use a ``multi`` link if:
+
+- The relationship is relatively stable and thus not updated very frequently.
+  For example, a list of postal addresses in a user profile.
+- The number of elements in the link tends to be small.
+
+Otherwise, prefer a single link from one object coupled with a computed
+backlink on the other:
```

Comment:
I think we can fold the prior sentence from this change into this same paragraph. Otherwise it's weird that we say "Generally ``single`` links are recommended when modeling 1:N relations." But then go on to caveat it further.

What about:

```suggestion
Otherwise, prefer a single link from one object coupled with a computed
backlink on the other. This is marginally more efficient and generally
recommended when modeling 1:N relations:
```

And remove that sentence from lines 277:278?

Reply:
Ah yes, I was thinking about that a bit and looks like a good solution to the small duplication.

Reply:
Okay, deleted the other sentence now. Looks nice